### ISSUE TITLE
Missing label or instruction on input

### ISSUE THREAD

**Issue by johnnyblasta** [origo-map/lmsearch-plugin#56]
A label or intructions are missing on the search input, see WCAG
[WCAG](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=332#labels-or-instructions)

Could be solved with a label infront of input or exchange spyglass icon with text button.
